### DISCUSSION TITLE
Whatsapp Revanced

### DISCUSSION THREAD

**Discussion by NinjaSurr** [Jarvis-Ank/Re-Vanced#25] [Ideas]
Can you make a whatsapp revanced? Because unofficial whatsapp mod is not safe so downloading whatsapp from playstore and patching it would help alot its okay even if its for root only users

**Comment by Jarvis-Ank** [comment]
First of all, I don't make patches for apps, it's done by the Revanced team, all i did was to make script to build it online.
2. Revanced team is currently focused on making AD removal/ premium patches for apps. WhatsApp mod is somewhat different from what they're working on rn, it's a complete mod, like new features and stuff. They may work on that in the future, like they did for Spotify, reddit, Twitter etc. You can submit your feature suggestion in the official Revanced page.
